homeassistant/components/tuya/sensor.py

```diff
@@ -595,5 +695,11 @@ def native_value(self) -> StateType:
         ):
             return None
 
+        # Get subkey value from Json string.
+        if self._status_range.type == "Json":
+            if self.entity_description.subkey is None:
+                return None
+            return json.loads(value).get(self.entity_description.subkey)
```

Comment:
This should be implemented as a DataType in `tuya/base.py`.

Reply:
That is a good question. Haven't really thought about that. I guess there are multiple approaches to this.

It could be defined as `ElectricityDataType`, as we know what we expect at this point and thus can be specific in the fields and its types.